src/Infrastructure/LeanCode.ExternalIdentityProviders/Apple/AppleIdService.cs

```diff
@@ -28,8 +28,8 @@ public AppleIdService(HttpClient httpClient, AppleIdConfiguration config, IMemor
             this.memoryCache = memoryCache;
 
             // https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/issues/1302
-            cryptoProviderFactory = new CryptoProviderFactory { CacheSignatureProviders = false };
-            tokenHandler = new JwtSecurityTokenHandler();
+            cryptoProviderFactory = new() { CacheSignatureProviders = false };
```

Comment:
Hmm, I'm not sure if it makes things readable. Maybe target typed new right near the field declaration?

Reply:
Actually, yeah, I could static–init those two. Will do.

Reply:
Done, the types should be obvious now.